tools/report-converter/tests/unit/sparse_output_test_files/sample.out

```diff
@@ -0,0 +1,5 @@
+files/sample.c:4:1: warning: symbol 'machine_id' was not declared. Should it be static?
+files/sample.c: note: in included file:
+./files/sample.h:3:40: warning: incorrect type in argument 1 (different address spaces)
+./files/sample.h:3:40:    expected struct task_struct *p1
+./files/sample.h:3:40:    got struct spinlock [noderef] __rcu *
```

Comment:
In my previous review comments I asked what is the reason that one line starts with `./` and the others are not (https://github.com/Ericsson/codechecker/pull/3160#discussion_r581055342).
Now with the regexes we will identify every line which stars with `./` as notes and the others as reports. Are you sure that every note line will start with `./`?

Reply:
Yes as far as I have checked, all the **note** lines start with `./`